### ISSUE TITLE
Tab closes too quickly

### ISSUE THREAD

**Issue by wes-novack** [seanstar12/zoom-close#7]
Hello! First, my system specs.

```
Fedora 31 (Workstation Edition)
Linux 5.5.13-200.fc31.x86_64 #1 SMP Wed Mar 25 21:55:30 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
Google Chrome Version 68.0.3440.75 (Official Build) (64-bit)
Zoom version: 3.5.385850.0413
Zoom Closer 1.6
```

When I have the Zoom Closer Chrome Extension enabled, it often closes the "Zoom Post Attendee window" BEFORE the tab has a chance to actually make the call to the Zoom client to launch the meeting.

So the meeting tab gets closed and I don't get put into the meeting.

Perhaps a configurable setting or delay could be enabled on the extension to allow for systems that need to keep the tabs open longer than others so that it actually works as intended?

**Comment by wes-novack** [comment]
Submitted a PR to increase the hardcoded timeout https://github.com/seanstar12/zoom-close/pull/8

I expect this should solve the issue.

**Comment by seanstar12** [comment]
I'll push the changes to the chome extension store this afternoon.

**Comment by wes-novack** [comment]
Thank you!

**Comment by wes-novack** [comment]
Resolved via PR #8 merged and released as version 1.7 in the chrome web store.
